### DISCUSSION TITLE
X-axis problem

### DISCUSSION THREAD

**Discussion by Ridegdog** [arkypita/LaserGRBL#853] [Q&A]
Hi i can burn a straight line with Y-axis but when i try the same line with X-axis i get this result. I try wih vectorise option. I dont know what is the problem please help.
![Image](https://i.imgur.com/XNK6VXz.jpg)

**Comment by StuartB4** [comment]
If your machine has belt drive make sure they are all tight, but also make sure the toothed pulley wheels on the motor shafts are tight.

**Comment by Ridegdog** [comment]
Okay now the lines are straight, but the start and the end point is not the same 
[https://i.imgur.com/5nZq4Iu.jpg](https://i.imgur.com/5nZq4Iu.jpg)
